Content.Client/DeltaV/Overlays/UltraVisionSystem.cs

```diff
@@ -16,16 +19,27 @@ public override void Initialize()
         SubscribeLocalEvent<UltraVisionComponent, ComponentInit>(OnUltraVisionInit);
         SubscribeLocalEvent<UltraVisionComponent, ComponentShutdown>(OnUltraVisionShutdown);
 
+        Subs.CVar(_cfg, DCCVars.NoVisionFilters, OnNoVisionFiltersChanged);
```

Comment:
What is `Subs`? Doesn't `_cfg` have a subscribe method you're supposed to use?

Reply:
Subs is new and preferred over _cfg

https://github.com/space-wizards/space-station-14/pull/25131